packages/quantic/README.md

```diff
@@ -31,6 +31,7 @@ Optionally install the [VSCode Salesforce Extension Pack](https://marketplace.vi
 ### Enable Dev Hub in Your Salesforce Organization
 
 - Connect to your salesforce developer organization [login](http://login.salesforce.com/)
+- Switch to Salesforces Classic.
```

Comment:
```suggestion

```

Reply:
I couldn't find the option to enable it in the setup in SF Lightning, though

Reply:
Nevermind, I can see it now... I must have made a typo or something hehe